### PR TITLE
Port SFS views and add FHIR details

### PR DESCRIPTION
**Pull request by joverlee521**
This PR is built on top of #26 and is dependent on https://github.com/seattleflu/id3c/pull/96.

1. Ports SFS specific views from core ID3C
2. Adds the details from `shipping.fhir_encounter_details_v1` to the modeling views. It currently coalesces `encounter_responses` and `fhir` to  be backwards compatible with the Audere format from Y1.

### CODE REVIEW

**Comment by joverlee521** [comment]
[Update modeling views to account for new encounter details (from FHIR ETL)](https://trello.com/c/9ZuXsq9D/285-update-modeling-views-to-account-for-new-encounter-details-from-fhir-etl)

**Comment by kairstenfay** [comment]
When I run `sqitch deploy` here (after running `sqitch deploy --log-only` on the related id3c branch), I get the following output:
```
$ sqitch deploy
Deploying changes to dev
  + shipping/views @2020-01-10 .. psql:schema/deploy/shipping/views@2020-01-10.sql:51: NOTICE:  view "metadata_for_augur_build_v1" does not exist, skipping
ok
  + shipping/views @2020-01-13 .. psql:schema/deploy/shipping/views@2020-01-13.sql:51: NOTICE:  view "metadata_for_augur_build_v1" does not exist, skipping
ok
  + shipping/views @2020-01-14 .. psql:schema/deploy/shipping/views.sql:51: NOTICE:  view "metadata_for_augur_build_v1" does not exist, skipping
ok
```
Is this expected? Should we push a separate commit to remove the code for the old `metadata_for_augur_build_v1` view?

**Comment by joverlee521** [comment]
> Is this expected? Should we push a separate commit to remove the code for the old metadata_for_augur_build_v1 view?

Ah, yeah. I didn't bother removing it because it wasn't doing anything. If the loud warning about it is annoying, we can remove it.

**Comment by joverlee521** [comment]
I've rebased on to master and removed the drop view line for `metadata_for_augur_build_v1 `

File: schema/deploy/shipping/views.sql [hunk 1]
```diff
@@ -487,8 +489,8 @@ create or replace view shipping.incidence_model_observation_v3 as
 
            residence_census_tract,
 
-           encounter_responses.flu_shot,
-           encounter_responses.symptoms,
+           coalesce(encounter_responses.flu_shot,fhir.vaccine) as flu_shot,
+           coalesce(encounter_responses.symptoms,fhir.symptoms) as symptoms,
```
Comment (tsibley) [inline schema/deploy/shipping/views.sql]: In the future we may want to flip the coalesce here to prioritize the FHIR responses.

**Comment by joverlee521** [comment]
Rebased on master to solve merge conflict.

**Pull request merged**